recipes/openassetio/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.0.0-alpha.7":
```

Comment:
That should be do-able - alpha.8 was released in parallel to the development of this recipe, so that's the only reason it's pinned to a slightly older release. I'll bump it and see what happens ;)

Reply:
Updated in ef8fea38